<launch>

    <node pkg="yolov4_trt_ros" type="trt_yolo_v4" name="yolov4_trt" output="screen" launch-prefix="valgrind --tool=callgrind --callgrind-out-file='callgrind.listener.%p' ">
        <param name="topic_name" value="/video_source/raw"/>
        <param name="model" value="yolov4"/>
        <param name="model_path" value="$(find yolov4_trt_ros)/yolo/"/>
        <param name="input_shape" type="string" value="416"/>
        <param name="category_number" type="int" value="80"/>
        <param name="confidence_threshold" type="double" value="0.5" />
        <param name="show_image" type="bool" value="True" />
    </node>
</launch>
